tasks.py

```diff
@@ -585,3 +585,10 @@ def compilemessages(ctx):
             "../dockerpythonvenv/bin/python manage.py compilemessages",
             **PLATFORM_ARG,
         )
+
+
+@task(aliases=["staging-to-review-app"])
+def staging_db_to_review_app(ctx, review_app_name):
```

Comment:
Can we add a simple docstring in here with the description and your instructions? This way it would be easy to lookup the syntax for the command it with an `inv -l`

Reply:
Absolutely! Thanks for the comment Rob!